CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 ## UNRELEASED
 
 IMPROVEMENTS:
+
 * Helm:
   * Add a `global.extraLabels` stanza to allow setting global Kubernetes labels for all components deployed by the `consul-k8s` Helm chart. [[GH-1778](https://github.com/hashicorp/consul-k8s/pull/1778)]
   * Add the `accessLogs` field to the `ProxyDefaults` CRD. [[GH-1816](https://github.com/hashicorp/consul-k8s/pull/1816)]
   * Add the `envoyExtensions` field to the `ProxyDefaults` and `ServiceDefaults` CRD. [[GH-1823]](https://github.com/hashicorp/consul-k8s/pull/1823)
   * Add the `balanceInboundConnections` field to the `ServiceDefaults` CRD. [[GH-1823]](https://github.com/hashicorp/consul-k8s/pull/1823)
+  * Add `HOST_IP` environment variable to mesh-gateway deployment
```

Comment:
Thank you for adding this, but let's remove it from this PR before it gets reviewed. We'll add it into the CHANGELOG later.

Reply:
Removed